static/beta/stage/modules/fed-modules.json

```diff
@@ -980,5 +980,22 @@
                 ]
             }
         ]
+    },
+    "trust": {
+        "manifestLocation": "/apps/trust/fed-mods.json",
+        "modules": [
+            {
+                "id": "trust",
+                "module": "./RootApp",
+                "routes": [
+                    {
+                        "pathname": "/inventory"
```

Comment:
This should be a full pathname to the location where the module should be mounted. I don't think `/insights` is the place. What is the URL to the page where these modules should be available at?